build.gradle

```diff
@@ -90,15 +90,6 @@ subprojects {
     }
   }
 
-  tasks.register('emptySourcesJar', Jar) {
-    // TODO: fetch sources from the corresponding AOSP repos.
-    archiveClassifier = 'sources'
-  }
-
-  tasks.register('emptyJavadocJar', Jar) {
-    archiveClassifier = 'javadoc'
-  }
-
```

Comment:
obsoleted by migration to Google's layoutlib (we used the prebuilt binaries, and therefore had no sources/javadocs artifacts).